P1/Project_Code/frontend/src/Theme.styles.jsx

```diff
@@ -4,13 +4,14 @@ import { createTheme } from "@mui/material";
 
 const theme = createTheme({
     // breakpoints: {
-// 	values: {
-// 		xs: 0,
-// 		sm: 375,
-// 		md: 786,
-// 		lg: 1440,
-// 		xl: 1536,
-// 	},
+	// values: {
```

Comment:
Are these comment neccessary? Are these breakpoint values being referenced in the application?